src/components/CalendarDay.js

```diff
@@ -10,7 +10,9 @@ const CalendarDay = ({day, isToday, events}) => {
     });
 
     return (
-        <div className={dayClasses}>
+      <div 
+        onClick={onClick.bind(null, 'day', day)}
```

Comment:
Haven't thought that through completely, but one way that should work (maybe not the prettiest) would be:

```js
const eventsForDay = (events || []).map(eventComp => eventComp.props.eventData);
```